pipeline/scripts/3-averageTechReplicates.R

```diff
@@ -122,3 +124,61 @@ repl.pattern.filtered <- retVal$pattern
 save(repl.pattern.filtered, file=paste(outdir, "repl.pattern.negative.RData", sep="/"))
 write.table(remove_neg, file=paste(outdir, "miss_infusions_neg.txt", sep="/"), row.names=FALSE, col.names=FALSE ,sep= "\t")
 
+# New: generate TIC plots
+run_name <- basename(outdir)
+TIC_input_dir <- paste(outdir, "2-pklist", sep = "/")
+
+# get replication pattern
+load(paste(outdir, "logs", "init.RData", sep="/"))
+# get misinjections
+bad_pos <- read.table(paste(outdir, "miss_infusions_pos.txt", sep="/"))
+bad_neg <- read.table(paste(outdir, "miss_infusions_neg.txt", sep="/"))
+
+# get all txt files
+TIC.files = list.files(TIC_input_dir, full.names=TRUE, pattern="*TIC.txt")
+all_samps <- sub('_TIC\\..*$', '', basename(TIC.files))
+
+print("\n")
+print(paste0("reading TIC files", TIC.files[1], " - till - ", TIC.files[length(TIC.files)]))
+# determine maximum intensity
+highest_tic_max <- 0
+for (file in TIC.files) {
+  tic <- read.table(file)
+  this_tic_max <- max(tic$TIC)
+  if (this_tic_max > highest_tic_max) {
+    highest_tic_max <- this_tic_max
+    max_sample <- sub('_TIC\\..*$', '', basename(file))
+    }
+}
+
+tic_plot_list <- list()
+k = 0
+for (i in c(1:length(repl.pattern))) { # change after test-phase !!!
```

Comment:
This comment is a bit confusing for a release branch in my opinion. Also, to what should it be changed?

Reply:
Comment removed.